bigbone/src/main/kotlin/social/bigbone/api/method/admin/AdminDomainAllowMethods.kt

```diff
@@ -0,0 +1,70 @@
+package social.bigbone.api.method.admin
+
+import social.bigbone.MastodonClient
+import social.bigbone.MastodonRequest
+import social.bigbone.Parameters
+import social.bigbone.api.Pageable
+import social.bigbone.api.Range
+import social.bigbone.api.entity.admin.AdminDomainAllow
+
+/**
+ * Allow certain domains to federate.
+ * @see <a href="https://docs.joinmastodon.org/methods/admin/domain_allows/">Mastodon admin/domain_allows API methods</a>
+ */
+class AdminDomainAllowMethods(private val client: MastodonClient) {
+
+    private val adminDomainAllowEndpoint = "api/v1/admin/domain_allows"
```

Comment:
Could you please rename this to `endpoint` only? I’ve started doing that in #410 for all endpoint references.

Reply:
I did not notice that you had made a PR based on my issue. I am glad it's been a good observation

Reply:
Sure thing, thanks for bringing it up! 😊